### DISCUSSION TITLE
docker compose up take a long time

### DISCUSSION THREAD

**Discussion by lovehoumin** [apache/airflow#45067] [Q&A]
I try to run `docker compose up -d` 

However, the step take for a long time...
![image](https://github.com/user-attachments/assets/c673eb3e-1844-49f9-9b96-05f03dbb0cdf)

Then I see the `docker ps -a` to show contaier status.
The init step already over one hour...
![image](https://github.com/user-attachments/assets/36684b47-cfb1-4961-ad0f-3b153d268245)

I try to see the log but it show empty...
![image](https://github.com/user-attachments/assets/38cd8d23-4159-4317-9c83-49a95f432013)

How to check the init status?
and how to speed up the init to start the other services.

**Comment by andresitorusz** [comment]
I faced the same issue last month. It turns out that bcoz I have dependency errors, one of my dependencies `pandas==2.0.3` is not supported on Python3.12 yet. I downgraded my Python version and everything is working.
